configure.ac

```diff
@@ -114,7 +114,12 @@ AS_IF([test "x$enable_unit" != xno], [
     AC_CHECK_PROG([XXD], [xxd], found, missing)
     AS_IF([test $XXD = missing],
           [AC_MSG_ERROR([Required executable xxd not found, some system tests will fail!])])
-    enable_unit="$enable_unit (tpm2_abrmd: $tpm2_abrmd, tpm_server: $tpm_server, bash: $BASH_SHELL, python: $PYTHON, xxd: $XXD)"
+
+    AC_CHECK_PROG([NETSTAT], [netstat], found, missing)
+    AS_IF([test $NETSTAT = missing],
+          [AC_MSG_ERROR([Required executable netstat not found, some system tests will fail!])])
+
+    enable_unit="$enable_unit (tpm2_abrmd: $tpm2_abrmd, tpm_server: $tpm_server, bash: $BASH_SHELL, python: $PYTHON, xxd: $XXD, netstat: $NETSTAT)"
 ])
```

Comment:
netstat is deprecated in favour of ss (from the iproute2 package) [according to its man page](http://net-tools.sourceforge.net/man/netstat.8.html), so  we should use that instead.